docs/user/accumulo/install.rst

```diff
@@ -57,25 +57,6 @@ The version of the distributed runtime JAR must match the version of the GeoMesa
 data store client JAR (usually installed in GeoServer; see below). If not,
 queries might not work correctly or at all.
 
-.. _install_accumulo_runtime_manual:
```

Comment:
Deleting this is probably the right call.  That said, lib/ext based installs are still available/possible on 1.9.x, right?

Reply:
yeah, i think it actually works on 2.0 too